src/index.html

```diff
@@ -4,19 +4,128 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__container">
+          <a href="index.html">
+            <img
+              src="./images/logo.png"
+              alt="logo"
+            />
+          </a>
+          <nav class="header__nav">
+            <ul class="header__menu">
+              <li class="menu-item">
+                <a
+                  href="#"
+                  class="link link-1 is-active"
+                >
+                  Apple
+                </a>
+              </li>
+              <li class="menu-item">
```

Comment:
Add a blank line between the links
```suggestion
              </li>
              
              <li class="menu-item">
```